setup.py

```diff
@@ -5,26 +5,45 @@
 import Xlib
 
 
-setup(name='python-xlib',
-        version=Xlib.__version_string__,
-
-        description='Python X Library',
-        download_url='https://github.com/python-xlib/python-xlib/releases',
-        url='https://github.com/python-xlib/python-xlib',
-        license='GPL',
-
-        author='Peter Liljenberg',
-        author_email='petli@ctrl-c.liu.se',
-
-        install_requires=['six>=1.10.0'],
-
-        packages=[
-            'Xlib',
-            'Xlib.ext',
-            'Xlib.keysymdef',
-            'Xlib.protocol',
-            'Xlib.support',
-            'Xlib.xobject'
-            ],
-        )
-
+setup(
+    name='python-xlib',
+    version=Xlib.__version_string__,
+
+    description='Python X Library',
+    download_url='https://github.com/python-xlib/python-xlib/releases',
+    url='https://github.com/python-xlib/python-xlib',
+    license='GPL',
```

Comment:
Contributors of the library agreed to change the license to LGPL v2.1 or later. See mailing list archive. I've still not replaced the license texts in .py files (only license.txt at the repo root folder) because we were not ready for release before your contributions.